.github/workflows/code_checks.yml

```diff
@@ -0,0 +1,61 @@
+on:
+  push:
+    branches:
+      - '**' # target all branches
+
+name: Static code checks
+jobs:
+  clippy_check_ubuntu:
+    name: Clippy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: rustup component add clippy
+
+      # Checks enabled everywhere, including tests, benchmarks
+      - uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does this exist? Looking at the repo I can't see it

Reply:
Nope. That's the wonder of copy/paste :D

Reply:
For some reason it doesn't work without it. I have no idea how CI works fully.